test/test_wrappers.jl

```diff
@@ -108,4 +108,32 @@ using StatsBase
             end
         end
     end
+    @testset "test adding and removing columns from Tables" begin
+        x, y = rand(10), rand(10)
+        tbl = Table(x=x, y=y)
+        mktempdir(pwd()) do tmp
+            path = joinpath(tmp, "tmp.lh5")
+            lh5open(path, "cw") do lhd
+                lhd["tbl"] = tbl
+                reduce_datastore(lhd, "tbl/y")
+                @test lhd["tbl"][:] == Table(tbl, y=nothing)
+                extend_datastore(lhd, "tbl", Table(y=y))
+                @test lhd["tbl"][:] == tbl
+            end
+        end
+    end
+    @testset "test adding and removing columns from NamedTuples" begin
+        x, y = rand(10), rand(10)
+        nt = (x=x, y=y)
+        mktempdir(pwd()) do tmp
+            path = joinpath(tmp, "tmp.lh5")
+            lh5open(path, "cw") do lhd
+                lhd["nt"] = nt
+                reduce_datastore(lhd, "nt/y")
```

Comment:
Same thing here for Windows (using `\\` instead of `/`).
Maybe this can be avoided using `joinpath(nt, y)`?

Reply:
`joinpath` didn't work...